public/pages/AnomalyCharts/containers/AnomalyDetailsChart.tsx

```diff
@@ -170,7 +170,8 @@ export const AnomalyDetailsChart = React.memo(
         zoomRange.endDate,
         taskId
       );
-      dispatch(searchResults(anomalyDataRangeQuery))
+      const resultIndex = get(props, 'detector.resultIndex', '');
```

Comment:
nit - you can move to the top of the file so it only needs to be declared once

Reply:
done